_includes/releases/v22.1/v22.1.7.md

```diff
@@ -0,0 +1,105 @@
+- Previously, escaping a double quote (`"`) with [`COPY`](../v22.1/copy-from.html) in `CSV` mode could ignore all subsequent lines in the same `COPY` if an `ESCAPE` clause were specified. This is now resolved. [#86977][#86977]
+- Fixed a bug that caused some special characters to be misread if they were being read by [`COPY ... FROM`](../v22.1/copy-from.html) into a `TEXT[]` column. [#86887][#86887]
+- Timescale object is now properly constructed from session storage, preventing  bugs and crashes in pages that use the timescale object when reloading the page [#86975][#86975]
+- Previously, CockroachDB would return an internal error when evaluating the `json_build_object` [builtin](../v22.1/functions-and-operators.html) when an [enum](../v22.1/enum.html) or a void datum were passed as the first argument, and this is now fixed. [#86851][#86851]
+- The statement tag for the [`SHOW`](../v22.1/show-vars.html) command results in the pgwire protocol no longer containing the number of returned rows. [#87126][#87126]
+- Fixed a bug where the options given to the [`BEGIN TRANSACTION`](../v22.1/begin-transaction.html) command would be ignored if the `BEGIN` was a prepared statement. [#87126][#87126]
+- A bug has been fixed that caused internal errors like "unable to [vectorize](../v22.1/vectorized-execution.html) execution plan: unhandled expression type" in rare cases. [#87182][#87182]
+- The **Explain Tab** inside the [Statement Details](../v22.1/ui-statements-page.html) page now groups plans that have the same shape but a different number of spans in corresponding scans. [#87211][#87211]
+- A bug in the column backfiller, which is used to add or remove columns from tables, failed to account for the need to read [virtual columns](../v22.1/computed-columns.html) which were part of a [primary key](../v22.1/primary-key.html). [Hash-sharded](../v22.1/hash-sharded-indexes.html) [indexes](../v22.1/indexes.html), starting in v22.1, use [virtual columns](../v22.1/computed-columns.html). Any [hash-sharded](../v22.1/hash-sharded-indexes.html) table created in v22.1 or any table created with a virtual column as part of its primary key would indefinitely fail to complete a [schema change](../v22.1/online-schema-changes.html) which adds or removes columns. This bug has been fixed. [#87272][#87272]
+- Added a missing memory accounting call when appending a KV to the underlying `kvBuf`. [#87118][#87118]
+- Some [upgrade](../v22.1/upgrade-cockroach-version.html) migrations perform [schema changes](../v22.1/online-schema-changes.html) on system tables. Those upgrades which added [indexes](../v22.1/indexes.html) could, in some cases, get caught retrying because they failed to detect that the migration had already occurred due to the existence of a populated field. When that happens, the finalization of the new version can hang indefinitely and require manual intervention. This bug has been fixed. [#87633][#87633]
```

Comment:
```suggestion
- Some [upgrade](../v22.1/upgrade-cockroach-version.html) migrations perform [schema changes](../v22.1/online-schema-changes.html) on system tables. Those upgrades which added [indexes](../v22.1/indexes.html) could, in some cases, get caught retrying because they failed to detect that the migration had already occurred due to the existence of a populated field. When that happens, the finalization of the new version could hang indefinitely and require manual intervention. This bug has been fixed. [#87633][#87633]
```

Reply:
Fixed!